map/src/drawer/util/LocalTrackUtils.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Looks very small function and probably doesn't need separate module

Reply:
`+`